src/pages/api/users/POST.ts

```diff
@@ -0,0 +1,19 @@
+import connectMongo from "../../../util/connectMongo";
+import User from "../../../../models/user";
+
+/**
+ * @param {import('next').NextApiRequest} req
+ * @param {import('next').NextApiResponse} res
```

Comment:
What is the purpose of using jsdoc syntax instead of typescript types? I see that the types for the `createUser` params are `any`, which seems to contradict the types you have supplied here.